src/Computation/SymbolicBitVector.cs

```diff
@@ -324,15 +330,5 @@ private IValue Create(IValue other, Func<Context, BitVecExpr, BitVecExpr, BoolEx
         {
             return new SymbolicBool(c => func(c, _func(c), other.ToSymbolicBitVector()._func(c)));
         }
-
-        private BigInteger ComputeInteger()
-        {
-            using var context = new Context();
```

Comment:
Is this to alleviate the memory pressure from creating many contexts?

Reply:
The actual alleviation of memory pressure is only really in the tests, where the `Context` is shared. The rest of this PR is really just to allow it to be passed in from outside, but that is a reasonable thing to do anyway.